### ISSUE TITLE
Code complete for font names doesn't interact well with quick editors

### ISSUE THREAD

**Issue by njx** [adobe/brackets-edge-web-fonts#3]
1. Trash prefs and open Edge Code (so you get the Getting Started project)
2. Put cursor in `<body>`
3. Cmd-E to bring up the quick editor
4. Click before the " in "Helvetica Neue" and hit Space
   -> No code hint appears, though it should.
5. Switch to main.css
6. Click before the " in "Helvetica Neue" and hit Space
   -> No code hint appears, though it should. (If you open main.css without having first touched the quick editor, it works.)
7. Switch back to index.html (which still has the quick editor open), set the cursor before "Helvetica Neue" again, and hit <space>
   -> Code hint menu appears at upper left of window, over the sidebar.

I'm guessing that code hints in general are probably broken with inline editors, but we haven't run into this before because we only have CSS/JS inline editors and up till now we've only had HTML code hints.

I'm also guessing that there are probably two bugs here--one having to do with the synchronization between the quick editor and the main editor, and one having to do with whatever figures out where to pop up the code hint menu.

Interestingly, if I hit Ctrl-Space in the inline editor (instead of Space), the menu pops up at the right location.

**Comment by njx** [comment]
Also, even if we get all this stuff working properly in quick editors, we'll still have the problem that the Wf button won't work while you're in the HTML file that contains the quick editor. I suppose we could do something like see if there are any quick editors open and gather the fonts from those.

**Comment by njx** [comment]
Actually, if focus is in a quick editor, we could just have the WF button generate the script tag based on the CSS file that editor is editing.

**Comment by njx** [comment]
adobe/brackets#1684 has a fix for these issues (everything except the Wf button not working when you're in an inline editor--I think that might be too much to fix at this point).

**Comment by njx** [comment]
That fix has been merged into Edge Code, and it fixes everything above except for #6--for some reason the code hint doesn't show in this case. (It does show if you type a space at the end of the line.) I'm digging into that now to see if it's some other problem.

**Comment by njx** [comment]
Turns out that's a totally unrelated problem, fixed by adobe/brackets#1685

**Comment by Zhuyun** [comment]
@njx 
Issue1: After step4, code hint appear on the left top corner on both MacOS X 10.8.1 and Win7.

Issue2: After step4, put cursor in 'h2' and Cmd+E to launch Quick Edit. Thus here bring up 2 quick edits, hit space in first one bring up correct hint, then hit space in the second bring up correctly again. Then back to the first one, hit space it brings up the hint of the name of the second.

Above issues both happened on English and French environment.
Screenshots:
 https://www.dropbox.com/s/e5jfb6uoy9qsj0q/QuickEdit%2BEWF_Issue.jpg?m

**Comment by njx** [comment]
@Zhuyun Have you pulled the latest Edge Code as well as EWF? The first issue you mention should be fixed.

**Comment by njx** [comment]
@Zhuyun Also, I don't have the test files you're using for your second issue, but it sounds like something that should be fixed by my fixes yesterday as well. Please make sure you have the latest version of both Edge Code and EWF and let me know if these still reproduce. Thanks.

**Comment by RaymondLim** [comment]
@Zhuyun If you're testing with b3788, then you don't have the fix yet. It is in Sprint 14 experimental build 0.14.0-3791.

**Comment by pthiess** [comment]
Added Sprint 14 label and FBNC

**Comment by Zhuyun** [comment]
Yesterday's latest for me is b3788. Now I just have b3791 and both issues fixed.

**Comment by joelrbrandt** [comment]
@Zhuyun has confirmed both issues fixed. Closing. Thanks!
